multisig-ism-metadata/src/main.sw

```diff
@@ -0,0 +1,72 @@
+library metadata;
```

Comment:
To be able to import it into a test contract (`multisig-ism-metadata-test`) so that I have access to everything in there directly. Testing via the functions exposed by `multisig-ism` wasn't really sufficient for full coverage